### PR TITLE
Gateway: Integration Playwright Test Stability.

### PR DESCRIPTION
**Pull request by sk-enya**
:clipboard:  
closes: https://github.com/bobanetwork/gateway/issues/582

Setting up integration test for bridging.
Screenshot 2024-07-13 at 10 28 54 PM

### CODE REVIEW

**Comment by souradeep-das** [comment]
not sure why the ci tests are failing?

**Comment by souradeep-das** [comment]
could we make the ci tests pass as well?

File: e2e-tests/README.md
Comment (souradeep-das) [inline e2e-tests/README.md:1]: nice!

**Review by boyuan-chen — APPROVED** [review]

**Review by souradeep-das — APPROVED** [review]

**Review by alvaro-ricotta — APPROVED** [review]
Looks good.

**Pull request merged**